erin_averill/first_vagrant_box/blog_post_message2/app/models/blog.rb

```diff
@@ -0,0 +1,10 @@
+class Blog < ActiveRecord::Base
+	has_many :owners
+	has_many :users, through: :owners
+	has_many :posts
+	has_many :messages, through: :posts
```

Comment:
Blog does not have many messages through posts(post is not the join table between message and blog) 
Missing.... has_many :user_posts, through: :posts, source: :user